src/pages-and-resources/discussions/app-config-form/index.js

```diff
@@ -1 +1 @@
-export { default } from './ConfigFormContainer';
+export { default } from './AppConfigForm';
```

Comment:
was that component renamed in some other commit?

Reply:
The component was moved into /app-config-form one commit and renamed from ConfigFormContainer to AppConfigForm in the next. (commits 3 and 4)

The PR is best-reviewed commit by commit...